src/shapes/Object/ObjectGeometry.ts

```diff
@@ -178,6 +178,10 @@ export class ObjectGeometry<EventSpec extends ObjectEvents = ObjectEvents>
     );
   }
 
+  isStrokeAccountedForInDimensions() {
+    return false;
+  }
```

Comment:
this isn't great as well but since @asturur yu do not want big refactors this is what I can offer

Reply:
The best thig i can think of is to kick the exactBoudingbox in its own polyline object.
I think long term makes more sense.
This seem unrelated from the PR entirely, can we remove it for now?

Reply:
I like this suggestion
Then we can mark the old one a legacy and set it in queue for deprecation or not.
It is related to the layout manager but indeed not to this PR.

Reply:
I will extract this commit to a dedicated PR